ui/pages/confirmation/confirmation.js

```diff
@@ -301,6 +308,47 @@ export default function ConfirmationPage({
     setApprovalFlowLoadingText(childFlow?.loadingText ?? null);
   }, [approvalFlows]);
 
+  useEffect(() => {
+    async function fetchSafeChainsList() {
+      try {
+        const response = await fetchWithCache({
+          url: 'https://chainid.network/chains.json',
+          cacheOptions: { cacheRefreshTime: DAY },
+          functionName: 'getSafeChainsList',
+        });
+        if (useSafeChainsListValidation) {
+          const safeChainsList = response;
+          const _matchedChain = safeChainsList.find(
+            (chain) =>
+              chain.chainId ===
+              parseInt(pendingConfirmation.requestData.chainId, 16),
+          );
+          setMatchedChain(_matchedChain);
+          if (
```

Comment:
@vthomas13 I think we should make this match even if cases are different. So change it to something like `_matchedChain?.nativeCurrency?.symbol?.toLowerCase() === pendingConfirmation.requestData.ticker?.toLowerCase()`

Reply:
(you pointed this out on our call last week)

Reply:
As we discussed, we'll handle this when refactoring the getAlerts function in add-ethereum-chain.js